common/source/docs/common-rangefinder-setup.rst

```diff
@@ -0,0 +1,42 @@
+.. _common-rangefinder-setup:
+
+===========================
+RangeFinders Setup Overview
+===========================
+
+There are many different kinds of rangefinders: Lidar (using laser or infra-red light to measure distance), Sonar (using ultrasonic sound), and Radar (using microwave RF). Some are analog, producing  pulses whose timing represent the distance to an object, others are digital sending data streams over serial to UARTs, or I2C, or even via UAVCAN.
+
+:ref:`RangeFinders (Sonar or Lidar) <common-rangefinder-landingpage>` can be used for :ref:`Object Avoidance <common-object-avoidance-landing-page>` in Copter-3.5 (and higher) in Loiter and AltHold modes and in Rover-3.5 (and higher). As well as altitude sensors for precision landing in Plane and Copter
+
+..  youtube:: y2Kk6nIily0
+    :width: 100%
+
+.. warning::
+   :ref:`common-object-avoidance-landing-page` features are new and should be used with caution.
+   
+
+Up to 10 Rangefinders can be used in the system. :ref:`Object Avoidance <common-object-avoidance-landing-page>` can use a single 360 degree Lidar, or up to 9 unidirectional rangefinders: 8 arranged in a circle covering 45 degree wide segments, plus an upward facing rangefinder. And a downward facing rangefinder used for low altitude height above ground measurements.
+
+Connecting and Configuring the Rangefinder
+==========================================
+
+- follow the instructions for each type of rangefinder described in its linked page on :ref:`common-rangefinder-landingpage`.
+- set the RNGFNDx_ORIENT parameters (i.e. :ref:`RNGFND1_ORIENT <RNGFND1_ORIENT>`, :ref:`RNGFND2_ORIENT <RNGFND2_ORIENT>`, etc.) to specify the direction each range finder is pointing in. 
+[site wiki="copter,rover"]
+  - For Copter and Rover, up to 8 rangefinders may placed around the vehicle to provide 360 degree coverage, or a single 360 degree Lidar. But only one forward facing rangefinder is required for :ref:`Object Avoidance <common-object-avoidance-landing-page>` . Since these may have obstructions in their field of view, so exclusion zones are provided via the PRX_IGN_ANGx and PRX_IGN_WIDx parameters, specifying a direction and width the frame obstruction presents and will be ignored. Up to 6 exclusion sectors can be specified.
+  - If a rangefinder is oriented facing up, then it will be used in the :ref:`common-simple-object-avoidance` operation as an upwards sensing proximity sensor in Copter LOITER mode only.
+[/site]
+  - If a rangefinder is oriented facing down, it will automatically be used to for height above ground measurements when within its range for increased landing precision in Copter and Plane autolandings.
```

Comment:
To be more specific for Copter, when landing the vehicle will slow it's descent to the LAND_SPEED (which defaults to 50cm/s) when it is within 10m of the ground according to the range finder.

Reply:
done